extensions/objects/models/objects.py

```diff
@@ -1709,3 +1709,22 @@ class TranslatableSetOfUnicodeString(BaseTranslatableObject):
         'contentId': None,
         'unicodeStrSet': [],
     }
+
+
+class ConvertCommaSepearatedToProperList(BaseObject):
```

Comment:
I don't think that the class name should explain the action, just the structure should be enough. So `CommaSeparatedString`

Reply:
Not done.